src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -621,8 +621,10 @@ AutoExifRotate =
 #    mkv      Matroska Video                                video/x-matroska
 #    mlp      Meridian Lossless Packing                     audio/vnd.dolby.mlp
 #    mov      QuickTime File Format                         video/quicktime
+#    mp2      MPEG Audio Layer II                           audio/mpeg
```

Comment:
@Nadahar I was just saying to add this as a comment **(video/vnd.dlna.mpeg-tts)** on the mpegts line, as an easy reminder for users.
Yes i tested it, and read of that related to rewind/forrward issues as well, that's said most af the time it's because UMS don't use the correct DLNA FLAGS...